app/templates/bekk/main.html

```diff
@@ -0,0 +1,222 @@
+{% extends "base.html" %} {% import "_form.html" as wtf %} {% block scripts %} {{ super() }}
+
+<script
+  src="{{url_for('static', filename='js/bekk/bekk.js') }}"
+  defer
+></script>
+
+{% endblock %} {% block app_content %}
+
+<div class="container mb-3">
+  <div class="btn-toolbar justify-content-between">
+    <h4>{{title}}</h4>
+    {% if has_permission(PermissionType.write, PermissionScope.bekk) %}
+    <div
+      data-bs-toggle="tooltip"
+      data-bs-placement="right"
+      title="{{ _('Ajouter un bekk') }}"
+    >
+      <button
+        type="button"
+        class="btn btn-success"
+        data-bs-target="#mo-bekk-add"
+        data-bs-toggle="modal"
+      >
+        <svg
+          class="bi flex-shrink-0"
+          width="24"
+          height="24"
+          role="img"
+        >
+          {{ macros.bootstrap_icon("plus-circle") }}
+        </svg>
+      </button>
+    </div>
+    {% endif %}
+  </div>
+</div>
+
+<div class="row">
+  <p>
+    {{ _("Bienvenue sur la page du Bekk de l'ESPCI.
+    <br />
+    Le Bekk est l'association journalistique de l'école, et produit plusieurs numéros par an, à des occasions et des
```

Comment:
plutôt séparer ça en deux traductions quand même, on évite trop d'HTML quand on peut

```suggestion
    {{ _("Bienvenue sur la page du Bekk de l'ESPCI.") }}
    <br />
    {{ _("Le Bekk est l'association journalistique de l'école, et produit plusieurs numéros par an, à des occasions et des
```